x-pack/metricbeat/module/statsd/server/mapping.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package server
+
+type StatsdMapping struct {
+	Metric string
+	Labels []Label
+	Value  Value
```

Comment:
Umm, is this correctly parsing these configs without the `config:..` annotations?

Reply:
yes, because the names of the field match the name in the config (lowercased)